doc/source/handling-dependencies.rst

```diff
@@ -254,7 +254,7 @@ API Reference
 
 The ``runtime_env`` is a Python dictionary including one or more of the following fields:
 
-- ``working_dir`` (str): Specifies the working directory for the Ray workers. This must either be an existing directory on the local machine with total size at most 100 MiB, or a URI to a remotely-stored zip file containing the working directory for your job. See :ref:`remote-uris` for details.
+- ``working_dir`` (str): Specifies the working directory for the Ray workers. This must either be (1) an local existing directory, (2) a local existing zipped file with total size at most 100 MiB, or (3) a URI to a remotely-stored zip file containing the working directory for your job. See :ref:`remote-uris` for details.
```

Comment:
Right now it looks like only (2) has the 100Mib limit, but can you clarify that it applies to both (1) and (2)?